src/Updater.rei

```diff
@@ -1,5 +1,4 @@
-type t('msg, 'model) = ('model, 'msg) => ('model, Effect.t('msg));
+type t('model, 'msg) = ('model, 'msg) => ('model, Effect.t('msg));
```

Comment:
This is a subtle breaking change which might produce some hard-to-understand errors, but I found it really confusing that this type, and only this type, had the order of type parameters reversed form every other type and even the update function itself.